frontend/src/scenes/pipeline/hogfunctions/templates/hog-templates.tsx

```diff
@@ -0,0 +1,10 @@
+import { HogFunctionTemplateType } from '~/types'
+
+export const HOG_FUNCTION_TEMPLATES: HogFunctionTemplateType[] = [
+    {
+        id: 'template-1',
+        name: 'Hello world',
+        description: 'Prints "Hello, World!" to the console.',
+        hog: `print "Hello, World!"`,
+    },
+]
```

Comment:
Would be lovely to have some real templates, cause the syntax errors here will trip anyone up who doesn't know Hog too well.

Reply:
Yeah thats all follow up

Reply:
Makes sense, but please just do `print('Hello World');` for v0.1... so it doesn't error immediately.

Reply:
I swapped it out for the fetch template as thats anyways the main use case